src/apps/activities/router.py

```diff
@@ -104,3 +108,15 @@
         **DEFAULT_OPENAPI_RESPONSE,
     },
 )(applet_activities_for_subject)
+
+router.get(
+    "/applet/{applet_id}/target/{target_subject_id}",
```

Comment:
This parameter name needs to align with the parameter consumed by the function, else the request looks like this in Swagger UI:

`http://127.0.0.1:8000/activities/applet/8e9bd4f9-7ea3-4cfd-b6b1-040108774370/target/%7Btarget_subject_id%7D?subject_id=8cb91d16-f1f2-48a8-bbf2-db2989c73cb4`

I'd vote to stick with `subject_id` since it means fewer additional request type definitions on the FE (can reuse the same one for this endpoint and the upcoming one you're assigned to).
```suggestion
    "/applet/{applet_id}/target/{subject_id}",
```

Reply:
Thanks for catching 🙏 I went with `subject_id`